resources/pdns_recursor_service_sysvinit.rb

```diff
@@ -41,8 +41,17 @@
 
 
 action :enable do
-  # Some sysvint systems start the default pdns-recursor after installations
-  # We make sure to disable it which also stops the service in some platforms
+
+  # Some distros start pdns-recursor after installing it, we want to stop it
+  # The behavior of the init script on CentOS 6 causes a bug so we skip it there
+  # (see https://github.com/dnsimple/chef-pdns/issues/77#issuecomment-311644973)
+  service 'pdns-recursor' do
```

Comment:
Cosigned about that. Resource cloning has been deprecated for some time in Chef, technically since just before Chef 11. While it is technically still supported via Chef 13, the side-affect is that you'll have the same resource being called multiple times in different ways which can make it unpredictable. See https://docs.chef.io/deprecations_resource_cloning.html about this behavior.